stat/stat.go

```diff
@@ -0,0 +1,83 @@
+package stat
+
+import (
+	"time"
+)
+
+// Method is an enumerator to describe [client.Client] methods.
+type Method int
+
+const (
+	MethodBalanceGet Method = iota
+	MethodContainerPut
+	MethodContainerGet
+	MethodContainerList
+	MethodContainerDelete
+	MethodContainerEACL
+	MethodContainerSetEACL
+	MethodEndpointInfo
+	MethodNetworkInfo
+	MethodObjectPut
+	MethodObjectDelete
+	MethodObjectGet
+	MethodObjectHead
+	MethodObjectRange
+	MethodSessionCreate
+	MethodNetMapSnapshot
+	MethodObjectHash
+	MethodObjectSearch
+	methodLast
```

Comment:
Do we have all client methods here? It should cover all of them even though the pool doesn't use some.

Reply:
Method list is updated, now we have all client methods